PROJECT_TIERS.md

```diff
@@ -0,0 +1,63 @@
+# Project Tiers
+
+The MapLibre Organization maintains the map rendering frameworks MapLibre GL JS and MapLibre GL Native and fosters an ecosystem of plugins and tools around these libraries to produce the best maps in the world.
+
+The three project tiers Core, Supported, and Hosted describe qualitatively how donated resources are distributed among the projects.
+
+## Tiers
+
+### Core Project
+
+A project that the MapLibre Organization is committed to in a long-term way. A substantial amount of funding is spent on the project in the form of paid Maintainer time and paid Bounties to ensure its long-term success.
+
+### Supported Project
+
+A project that the MapLibre Organization can allocate funding to in the form of paid Maintainer time and paid Bounties. Supported Projects receive a substantially smaller amount of funding than Core Projects and the MapLibre Organization cannot guarantee any level of long-term support.
```

Comment:
Only bounties?

Reply:
Reduced to Bounties now. Let us keep in mind though that Bounties will need some management work which someone will need to do as a volunteer.